src/components/Trade/hooks/useSwapper/utils.ts

```diff
@@ -79,11 +79,9 @@ export const getFirstReceiveAddress: GetFirstReceiveAddress = async ({
 export const getSelectedReceiveAddress: GetSelectedReceiveAddress = async ({
   chainAdapter,
   wallet,
-  buyAssetAccountId: accountId,
+  bip44Params,
 }) => {
-  // TODO accountType and accountNumber need to come from account metadata
-  const { accountType, utxoParams } = accountIdToUtxoParams(accountId, 0)
-  return await chainAdapter.getAddress({ wallet, accountType, ...utxoParams })
+  return await chainAdapter.getAddress({ wallet, bip44Params })
```

Comment:
careful, we need accountType here

Reply:
3c1753e7c8183e9d06440584e0bbd69997837461